src/main/java/pl/nogacz/snake/application/Achievement.java

```diff
@@ -0,0 +1,97 @@
+package pl.nogacz.snake.application;
+
+import java.awt.SystemTray;
+import java.awt.TrayIcon.MessageType;
+import java.awt.Image;
+import java.awt.Toolkit;
+import java.awt.TrayIcon;
+
+public abstract class Achievement {
+
+    protected int milestone;
+    public int id;
+
+    protected static final int max=10;
```

Comment:
eklediğin kodun genelinde parçalardan sonra birer boşluk bırak. Kodun geneliyle daha uyumlu olacak.

`max=10` => `max = 10`

Birçok yerde aynı durum söz konusu. Tek tek işaretlemeyeceğim. Sen yaptığın değişikliklere bak ve düzelt lütfen.

Reply:
Düzeltildi.